### PR TITLE
Feature/move remote data to image

### PR DESCRIPTION
**Pull request by fivejjs**
To avoid too request limits of github, update the configure to load local files in for k8s batch jobs.

### CODE REVIEW

**Comment by cbur24** [comment]
@fivejjs I removed the `dask_chunks` param from the `gm_mads_two_seasons_training` function, and restored the full string path for the slope variable as this function will only be used in notebooks for training models and not as an K8S workflow.

With the `Dockerfile` changes, does this mean the entire `Production` and `Testing` folders are copied to the Docker image?  If so, what is the reason for copying the `Production` folder? Since all the item (geojsons, models) are in the `Testing` folder?

File: Dockerfile [hunk 1]
```diff
@@ -34,6 +34,10 @@ RUN apt-get update \
     && sed 's/#.*//' /tmp/apt-run.txt | xargs apt-get install -y \
     && rm -rf /var/lib/apt/lists/*
 
+# Read configure from docker image folder
+COPY production /crop-mask/production
+COPY testing /crop-mask/testing
+
```
Comment (alexgleith) [inline Dockerfile:40]: Perhaps only include production? Testing is run on a small scale and it is probably more flexible not to have the assets static in the image.

**Review by alexgleith — CHANGES_REQUESTED** [review]
I think you should copy only the SPECIFIC assets you need into the image, rather than all the code in the `production` and `testing` folders.

Being explicit about what files you need is more mature and again, makes things more easily reproducible.

File: Dockerfile (hunk 1 above)
Comment (cbur24) [reply Dockerfile:40]: Don't be confused by the 'testing' terminology, the production models etc are stored in the testing folder as they are created using notebooks.

Comment (cbur24) [reply Dockerfile:40]: @fivejjs I agree that we should be explicit with the files in this case (even thogh I like the simplicity of just copying one folder).  Lets just copy the `td`, `model`, and `aez` files into the docker image.  Then when we run the other models for the other regions, I'll just keep adding new file paths to the `Dockerfile`

Comment (fivejjs) [reply Dockerfile:40]: sure.

Comment (fivejjs) [reply Dockerfile:40]: Some of the  files got the timestamp
```
/crop-mask/testing/eastern_cropmask/results/gm_mads_two_seasons_ml_model_20210427.joblib
```
We add `/crop-mask/testing/eastern_cropmask/results/gm_mads_two_seasons_ml_model_*`

File: Dockerfile [hunk 2]
```diff
@@ -34,6 +34,13 @@ RUN apt-get update \
     && sed 's/#.*//' /tmp/apt-run.txt | xargs apt-get install -y \
     && rm -rf /var/lib/apt/lists/*
 
+# Read configure from docker image folder
+COPY testing/eastern_cropmask/results/gm_mads_two_seasons_ml_model_* /crop-mask/testing/eastern_cropmask/results/
```
Comment (cbur24) [inline Dockerfile:38]: Do we have to copy the folders? Can't we copy just the `model`, `td`, and `aez` files themselves?

Comment (fivejjs) [reply Dockerfile:38]: the model all have datetime. Do you want to only copy one datetime?

**Review by alexgleith — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (fivejjs) [reply Dockerfile]: now only specific model file is copied.

**Pull request merged**